### PR TITLE
Fix UI navbar highlighting

### PR DESCRIPTION
**Pull request by eirslett**
The "Find a trace" menu element was hard-coded
to always being selected.

This fix creates a NavbarUI component, which
listens to "navigate" events and highlights the
correct menu item, depending on the route.

### CODE REVIEW

File: zipkin-web/src/main/resources/app/js/page/dependency.js [hunk 1]
```diff
@@ -24,22 +26,28 @@ define(
...
-      window.document.title = 'Zipkin - Dependency';
-      $('.content').html(dependenciesTemplate());
+        this.$node.html(dependenciesTemplate());
 
-      const {startTs, endTs} = queryString.parse(location.search);
-      $('#endTs').val(endTs || moment().valueOf());
-      $('#startTs').val(startTs || moment().valueOf() - 7*24*60*60*1000); // set default startTs 7 days ago;
+        const {startTs, endTs} = queryString.parse(location.search);
+        $('#endTs').val(endTs || moment().valueOf());
+        $('#startTs').val(startTs || moment().valueOf() - 7*24*60*60*1000); // set default startTs 7 days ago;
```
Comment (yurishkuro) [inline zipkin-web/src/main/resources/app/js/page/dependency.js:38]: is this the same behavior as before?  I thought the lookback interval was configurable via env variable.

Comment (eirslett) [inline zipkin-web/src/main/resources/app/js/page/dependency.js:38]: It seems to be configurable as a flag in zipkin-query. To be honest, I'm not 100 % sure what it does.

Comment (codefromthecrypt) [inline zipkin-web/src/main/resources/app/js/page/dependency.js:38]: So before Yuki made a change, we had no way to specify the interval of
dependencies query in the UI. If we want to poke a hole and default it from
env on the web or query server, I suppose that would be in the other config
endpoint?

Comment (codefromthecrypt) [inline zipkin-web/src/main/resources/app/js/page/dependency.js:38]: In general I think now is a good time to think of things like this as
config servers vs abstraction leaks (like JavaScript reading env values of
things). That's why I prefer the config endpoint approach as at least it
contains it until there's a better answer.

Comment (eirslett) [inline zipkin-web/src/main/resources/app/js/page/dependency.js:38]: Could somebody find a commit in the history (before I made the refactorings) with the Scala logic that sets it up? I'm might have overseen it.

Comment (codefromthecrypt) [inline zipkin-web/src/main/resources/app/js/page/dependency.js:38]: I'll look it up and ping you. This change made a historical one obvious and shouldn't be held back on account of moving it two spaces to the right.

**Pull request merged**